_docker/drupal/drupal-filesystem/web/themes/custom/rhd/css/base/eclipse.css

```diff
@@ -0,0 +1 @@
+.wide.wide-hero.eclipse{background:url(/images/eclipse/eclipse_microsite_hero.jpg);background-size:cover;height:450px}
```

Comment:
@KamiQuasi Its worth checking with @LightGuard whether this should be committed. I believe most of the CSS in this directory is generated, so not sure if this should be here permanently.

cheers,

Rob